packages/lib/mattermost/service.ts

```
@@ -0,0 +1,137 @@
+
+    nextCursor = data.response_metadata?.next_cursor;
+  } while (nextCursor);
+
+  return channels;
+};
+
+export const getMattermostChannels = async (environmentId: string): Promise<TIntegrationItem[]> => {
+  let channels: TIntegrationItem[] = [];
+  try {
+    const slackIntegration = (await getIntegrationByType(environmentId, "slack")) as TIntegrationSlack;
+    if (slackIntegration && slackIntegration.config?.key) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)